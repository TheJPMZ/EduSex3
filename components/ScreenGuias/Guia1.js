import * as React from 'react';
import { Text, View, StyleSheet, Image, ScrollView } from 'react-native';

const Screen1 = ( {navigation} ) => {
  return (
    <>
      <View style={styles.container}>
        <Text style={styles.textTittle}>Anillo Vaginal</Text>
      </View>
      <View style={styles.container2}>
      </View>
    </>
  );
}

export default Screen1

const styles = StyleSheet.create({
  container: {
    alignItems: 'center',
    justifyContent: 'center',
    width : '100%',
    height : 150,
    marginBottom : -50,
    borderRadius : 15,
    backgroundColor : '#FFFFFF',
    overflow : 'hidden'
  },

  container2: {
    alignItems: 'center',
    justifyContent: 'center',
    width : '100%',
    height : '100%',
    padding: 30,
    borderRadius : 15,
    backgroundColor : '#FFFFFF'
  },

  textTittle : {
    fontWeight : 'bold',
    fontSize : 40
  },

  image : {
    width : '100%',
    height : '70%'
  },

  scrollView: {
    backgroundColor: 'transparent',
    marginHorizontal: 20,
    paddingTop: 10
  }
});