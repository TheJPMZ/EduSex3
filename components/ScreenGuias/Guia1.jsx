import * as React from 'react';
import {
  Text, View, StyleSheet,
} from 'react-native';
import { TouchableOpacity } from 'react-native-web';

const styles = StyleSheet.create({
  container: {
    alignItems: 'center',
    justifyContent: 'center',
    width: '100%',
    height: 150,
    marginBottom: 0,
    borderRadius: 15,
    backgroundColor: '#FFFFFF',
    overflow: 'hidden',
  },

  container2: {
    alignItems: 'center',
    justifyContent: 'center',
    width: '100%',
    height: '100%',
    paddingHorizontal: 30,
    borderRadius: 15,
    backgroundColor: '#FFFFFF',
  },

  textTittle: {
    fontWeight: 'bold',
    fontSize: 40,
  },

  image: {
    width: '100%',
    height: '70%',
  },

  scrollView: {
    backgroundColor: 'transparent',
    marginHorizontal: 20,
    paddingTop: 10,
  },

  containerText: {
    margin: 10,
    fontSize: 15,
    color: '#333333',
    marginBottom: 5,

  },
  button: {
    alignItems: 'center',
    backgroundColor: '#dd7973',
    padding: 20,
    paddingRight: 16,
    paddingLeft: 16,
    margin: 5,
    marginHorizontal: 20,
    borderRadius: 15,
  },
  buttonText: {
    fontSize: 20,
    color: '#000000',
  },
});

function Guia1({ navigation }) {
  return (
    <>
      <View style={styles.container}>
        <Text style={styles.textTittle}>Consentimiento</Text>
      </View>
      <View style={styles.container2}>
        <Text style={styles.containerText}>
          El consentimiento es el permiso que una persona da para
          que se realice una acción. En el contexto de desarrollo
          personal de una persona, lo que se entiende como
          consentimiento es el permiso que una persona da para que
          se realice una acción que involucre su cuerpo o su bienestar.
        </Text>
        <Text style={styles.containerText}>
          El consentimiento en el ámbito sexual se puede expresar
          de distintas maneras. Como el consentimiento a actos
          sexuales, a afecto físico e incluso a fotos explicitas.
        </Text>
        <Text style={styles.containerText}>
          Por lo tanto se deben tomar varias cosas en cuenta al
          empezar un acto sexual. Dentro de estas se pueden encontrar:
        </Text>
        <Text style={styles.containerText}>
          -El consentimiento se puede retirar en cualquier momento
        </Text>
        <Text style={styles.containerText}>
          - No se debe intentar convencer a obtener consentimiento sobre algo.
        </Text>
        <Text style={styles.containerText}>
          - Cuando alguien no esá lo suficientemente cuerdo o
          consciente para dar consentimiento no se puede aceptar el consentimiento dado.
        </Text>
        <Text>
          - Aprovecharse de una situación para obtener consentimiento está prohibido.
        </Text>
        <Text style={styles.containerText}>
          - Y lo más importante de todo... Si una persona dice NO, significa NO
        </Text>
        <TouchableOpacity
          style={styles.button}
          onPress={() => navigation.navigate('Preguntas 1')}
        >
          <Text style={styles.buttonText}>Evaluar</Text>
        </TouchableOpacity>
      </View>
    </>
  );
}

export default Guia1;
