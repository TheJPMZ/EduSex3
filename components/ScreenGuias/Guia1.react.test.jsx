import React from 'react';
import renderer from 'react-test-renderer';
import Guia1 from './Guia1';


test('renders correctly', () => {
  const tree = renderer.create(<Guia1 />).toJSON();
  expect(tree).toMatchSnapshot();
});



