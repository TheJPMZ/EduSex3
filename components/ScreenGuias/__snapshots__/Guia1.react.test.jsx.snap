// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
[
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 15,
        "fontSize": 15,
        "height": 150,
        "justifyContent": "center",
        "marginBottom": 0,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 40,
          "fontWeight": "bold",
        }
      }
    >
      Consentimiento
    </Text>
  </View>,
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 15,
        "height": "100%",
        "justifyContent": "center",
        "paddingHorizontal": 30,
        "width": "100%",
      }
    }
  >
    <RCTScrollView>
      <View>
        <Text>
          El consentimiento es el permiso que una persona da para que se realice una acción. En el contexto de desarrollo personal de una persona, lo que se entiende como consentimiento es el permiso que una persona da para que se realice una acción que involucre su cuerpo o su bienestar.
          


          El consentimiento en el ámbito sexual se puede expresar de distintas maneras. Como el consentimiento a actos sexuales, a afecto físico e incluso a fotos explicitas.
          


          Por lo tanto se deben tomar varias cosas en cuenta al empezar un acto sexual. Dentro de estas se pueden encontrar:
          


          -El consentimiento se puede retirar en cualquier momento
          


          - No se debe intentar convencer a obtener consentimiento sobre algo.
          


          - Cuando alguien no esá lo suficientemente cuerdo o consciente para dar consentimiento no se puede aceptar el consentimiento dado.
          


          - Aprovecharse de una situación para obtener consentimiento está prohibido.
          


          - Y lo más importante de todo... Si una persona dice NO, significa NO
        </Text>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#dd7973",
              "borderRadius": 15,
              "margin": 5,
              "marginHorizontal": 20,
              "opacity": 1,
              "padding": 20,
              "paddingLeft": 16,
              "paddingRight": 16,
            }
          }
        >
          <Text
            style={
              {
                "color": "#000000",
                "fontSize": 20,
              }
            }
          >
            Evaluar
          </Text>
        </View>
      </View>
    </RCTScrollView>
  </View>,
]
`;
